index.html

```diff
@@ -254,15 +254,15 @@ <h4>AVM 1</h4>
 					</p>
 					<p>
 					<h5>ActionScript 1 &amp; 2 Language <code>85%</code></h5>

```

Comment:
This number should be updated too (same for the others)

Reply:
Oops,  thanks.